src/reducers/MapReducer.js

```diff
@@ -19,42 +19,12 @@ import {campsite} from '../locale.en';
 
 const {campsite_form: {accessibility_options, facilities_options, features_options, price_options}} = campsite;
 
-const sampleSiteMarkers = [
-    {
-        id: 1,
-        title: 'Twin Lakes Hidden Spot',
-        description: 'Beautiful view of Twin Lakes off this hidden forest road.',
-        coordinate: {
-            longitude: -106.391015,
-            latitude: 39.085855
-        }
-    },
-    {
-        id: 2,
-        title: 'Lily Lake',
-        description: 'Nice view of the lilypads in this secluded spot, but a pretty tough road to reach it.',
-        coordinate: {
-            longitude: -106.368051,
-            latitude: 39.351661
-        }
-    },
-    {
-        id: 3,
-        title: 'Slide Lake',
-        description: 'Pretty riverside camping, but a REALLY nasty road to get there.',
-        coordinate: {
-            longitude: -106.389204,
-            latitude: 39.372171
-        }
-    }
-];
-
 const INITIAL_STATE = {
     lastKnownRegion: {
-        longitude: -105.727939,
-        latitude: 39.695168,
-        longitudeDelta: 1,
-        latitudeDelta: 1
+        latitude: 39.50650368421296,
+        latitudeDelta: 1.9622561912834797,
+        longitude: -105.97962943486341,
+        longitudeDelta: 1.721545855432879
```

Comment:
Updated our initial region for first login to be one that highlights the existing sites we already have in the app.